docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+services:
+  web:
+    ports:
+     - "4000:4000"
+    volumes:
+    - $HOME/src/openshift-playbooks:/opt/jekyll
```

Comment:
Instead of `$HOME/src/openshift-playbooks:/opt/jekyll`, can we reference `./:/opt/jekyll`